src/pages/Profile/ProfileAside.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from 'styled-components';
+import { PROFILE_MENU_DATA } from './data';
+
+const ProfileAside = ({
+  goProfileFirstMenu,
+  goProfileSelectedMenu,
+  profileMenu,
+}) => {
+  return (
+    <ProfileAsideWrapper>
+      <MyProfile onClick={() => goProfileFirstMenu}>MY 필링</MyProfile>
```

Comment:
- 함수를 실행해 주셔야 제대로 작동할 것 같습니다.
```suggestion
      <MyProfile onClick={() => goProfileFirstMenu()}>MY 필링</MyProfile>
```

Reply:
리뷰 반영했습니다! 감사합니다!